### ISSUE TITLE
Curl import changes URL

### ISSUE THREAD

**Issue by allanf181** [usebruno/bruno#2379]
### I have checked the following:

- [X] I use the newest version of bruno.
- [X] I've searched existing issues and found nothing related to my issue.

### Describe the bug

When a curl command is imported, the url is changed by removing the last /
`curl 'http://example.com/test/'`  turns into `http://example.com/test`

This affects endpoints that end with /

### Screenshots/Live demo link

https://github.com/usebruno/bruno/assets/14101716/8c417d75-78f6-49ac-a008-737a63840656

**Comment by satokenta940** [comment]
When you import cURL from [Apidog](https://www.apidog.com/?utm_source=&utm_medium=blogger&utm_campaign=test1), you won't encounter this issue. Give it a try; I think it's quite effective.

**Comment by jwetzell** [comment]
@jokj624 that line is actually specifically written to remove the `/` from the end of a url. So seems like they are intentionally stripping of that last backslash for some reason. Not sure why the `raw_url` field also gets the processed value. That does not seem "raw" to me.

**Comment by jkjustjoshing** [comment]
I can't imagine a situation in which I want the request that Bruno sends out to be different from the request cURL sends out. If the user doesn't want a trailing slash they would just delete it from the URL after import (or in the cURL command).

This caused me a huge headache yesterday. I was trying to hit this Python FastAPI endpoint, and I was getting a correct response using cURL on the command line, but importing into Bruno kept giving me a "405 method not allowed" error.

![image](https://github.com/user-attachments/assets/4db457f8-2daf-402f-8292-462a6160ffe7)

I ended up spinning up a webserver that echoed back the headers, thinking maybe Bruno was attaching a hidden header somewhere? I thought it would be great if Bruno had an "export to cURL" feature, so I could have done a string comparison with the cURL syntax, but I don't believe that exists in Bruno.

15 minutes later, I realized that the trailing `/` was being stripped off.
